typescript/infra/scripts/funding/update-balance-threshold-config.ts

```diff
@@ -0,0 +1,105 @@
+import { checkbox } from '@inquirer/prompts';
+import yargs from 'yargs';
+
+import { ChainMap } from '@hyperlane-xyz/sdk';
+import {
+  LogFormat,
+  LogLevel,
+  configureRootLogger,
+  rootLogger,
+} from '@hyperlane-xyz/utils';
+
+import rawDailyBurn from '../../config/environments/mainnet3/balances/dailyBurn.json';
+import { readJSONAtPath, writeJsonAtPath } from '../../src/utils/utils.js';
+import { withBalanceThresholdConfig } from '../agent-utils.js';
+
+import {
+  BalanceThresholdConfig,
+  configFileNameMapping,
+  dailyBurnMultiplier,
+} from './utils/constants.js';
+import { THRESHOLD_CONFIG_PATH } from './utils/grafana.js';
+import { formatDailyBurn } from './utils/utils.js';
+
+const dailyBurn: ChainMap<number> = rawDailyBurn;
+
+export const configChoiceLabelMapping: Record<BalanceThresholdConfig, string> =
+  {
+    [BalanceThresholdConfig.RelayerBalance]: 'Desired Relayer Balance',
+    [BalanceThresholdConfig.LowUrgencyKeyFunderBalance]:
+      'Low Urgency Key Funder Balance Alert Threshold',
+    [BalanceThresholdConfig.LowUrgencyEngKeyFunderBalance]:
+      'Low Urgency Eng Key Funder Balance Alert Threshold',
+    [BalanceThresholdConfig.HighUrgencyRelayerBalance]:
+      'High Urgency Relayer Balance',
+  };
+
+async function main() {
+  configureRootLogger(LogFormat.Pretty, LogLevel.Info);
+  const { balanceThresholdConfig } = await withBalanceThresholdConfig(
+    yargs(process.argv.slice(2)),
+  ).argv;
+
+  const configToUpdate: BalanceThresholdConfig[] = balanceThresholdConfig
+    ? [balanceThresholdConfig]
+    : [];
+
+  if (configToUpdate.length === 0) {
```

Comment:
is there a downside to just always requiring that all configs are updated? can't think of a scenario where we wouldn't wanna update all of them just in case